### PR TITLE
fix local-up-volcano broken

### PR DESCRIPTION
**Pull request by hwdef**
fix: #2904

### CODE REVIEW

**Comment by lowang-bh** [comment]
LGTM

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Comment by xiao-jay** [comment]
LGTM，local-up-volcano.sh  should pull local image first,I agree with you @hwdef .
@wangyang0616 @william-wang please review this pr .

File: hack/local-up-volcano.sh [hunk 1]
```diff
@@ -41,7 +39,10 @@ function prepare {
 
 function install-volcano {
   # TODO: add a graceful way waiting for all crd ready
-  kubectl apply -f ${RELEASE_FOLDER}/${YAML_FILENAME}
+  kubectl create namespace volcano-system 
+  helm install volcano ${VK_ROOT}/installer/helm/chart/volcano --namespace volcano-system \
```
Comment (wangyang0616) [inline hack/local-up-volcano.sh:43]: [#2936](https://github.com/volcano-sh/volcano/pull/2936) has added depend information to Volcano's helm package. Before installing here, do you need to execute `helm dependency build` first?

Comment (hwdef) [reply hack/local-up-volcano.sh:44]: done

**Pull request merged**